### DISCUSSION TITLE
Mu to e gamma calculation in the B-L model using the light neutrinos only

### DISCUSSION THREAD

**Discussion by ahmost26** [FeynCalc/feyncalc#175] [Q&A]
Hello,

I was trying to calculate the amplitude of the mu to e gamma decay and of course there are four one loop diagrams that effectively  contributes to the magnetic transition. anyway, I am stuck at calculating the diagram that contains two W-bosons in the loop i tried many times calculating it and the final result is different than Cheng and Li book in several orders of magnitude, I tried to alter the momentum my self so that it resembles the book but also the answer i get is not right.

(g2 STW g^(\[Beta]\[Rho]) g^(\[Nu]\[Sigma]) \[CurlyEpsilon]^*^\[Mu](q) (g^(\[Mu]\[Sigma]) (-k+p1+2 q)^\[Beta]+g^(\[Beta]\[Sigma]) (2 k-2 p1-q)^\[Mu]+g^(\[Beta]\[Mu]) (-k+p1-q)^\[Sigma]) (\[CurlyPhi](p1,MassFe(1))).(-((I g2 \[Gamma]^\[Rho].Overscript[\[Gamma], _]^7 IndexSum(ZEL(1,j1) UV(1,j1)^*,{j1,3}))/Sqrt[2])).(\[Gamma]\[CenterDot]k+MassFv(1)).(-((I g2 \[Gamma]^\[Nu].Overscript[\[Gamma], _]^7 IndexSum(UV(1,j1) ZEL(2,j1)^*,{j1,3}))/Sqrt[2])).(\[CurlyPhi](p,MassFe(2))))/(k^2-MassFv(1)^2).(((k-p1))^2-MassVWm^2).(((k-p1-q))^2-MassVWm^2)

this is the generated amplitude from Fyncalc.
 and then i followed the steps to to make the amplitude in the PaVe form and the Pax-evaluate and i took this approximation to get the terms i want,
$Assumptions = {Element[{MassVWm, x, y}, Reals], y > 1, x > 1, 
   MassVWm > 1};
amp33 = PaXEvaluate[f1, k, PaXImplicitPrefactor -> 1/(2 Pi)^D, 
   PaXC0Expand -> True] /. {MassFe[2] -> Sqrt[x] MassVWm, 
   MassFv[1] -> Sqrt[y] MassVWm}
and then i expanded the series with the following command
 f3 = Normal[Series[amp6, {x, 0, 2}, {y, 0, 1}]] // FullSimplify
f4 = f3 /. {x -> MassFe[2]^2/MassVWm^2, y -> MassFv[1]^2/MassVWm^2} //
   FullSimplify

and then this give me the wrong answer and i dont know why.
there is an important fact that this method gave me the right answer of the diagram that has two Higgs in the loop and i got the same answer as the book of Cheng and Li but i dont know why it doesn't work in this loop.

Thank you.

**Comment by vsht** [comment]
Hi,

there are certainly people more knowledgeable than me on this topic (I hardly do any BSM),
but with the amount of details you provided it is not really easy to understand the issue.

Having a self-contained notebook with the working code and the relevant references is one thing, 
while bits of expressions here and there and a statement that something comes out wrong is
something totally different. Personally, I don't know the Cheng & Li book by heart and 
when you write "**the** B-L model" there is no Lagrangian popping up in front of my eyes.

So yeah, how about filing those gaps?

Cheers,
Vladyslav
